### PR TITLE
docs: Scala in action docs.

### PR DESCRIPTION
**Pull request by RayRoestenburg**
I need to rebase this on main, once 
- [x] #507, 
- [x] #515, and 
- [x] #513 
are merged. 
- [x] need to do the fibonacci sample first.
(now just pulled in changes from #507 to see if docs would generate)

### CODE REVIEW

File: docs/src/modules/java/pages/actions-as-controller.adoc [hunk 1]
```diff
@@ -20,21 +24,46 @@ include::example$java-valueentity-counter/src/main/proto/com/example/actions/dou
 <2> The protobuf option (akkaserverless.service) is specific to code-generation as provided by the Akka Serverless Maven plugin. This annotation indicates to the code-generation that an Action must be generated.
 <3> The Action mimics the Counter API
 
+[.tabset]
```
Comment (patriknw) [inline docs/src/modules/java/pages/actions-as-controller.adoc:27]: this `[.tabset]` should be removed, it should only be one at the top, and then `Java::` ... `Scala::`

File: docs/src/modules/java/pages/actions-publishing-subscribing.adoc [hunk 2]
```diff
@@ -32,6 +35,8 @@ include::example$java-eventsourced-counter/src/main/proto/com/example/actions/co
 
 The class `CounterJournalToTopicAction` gets generated for us based on the proto file defined above.
 
+ifdef::todo[TODO: EventSourced counter example Scala]
```
Comment (patriknw) [inline docs/src/modules/java/pages/actions-publishing-subscribing.adoc:38]: The way I did it in the views doc was to add a Scala tab, but inside it just have `TODO`. Might be confusing to only have Java tab? Not important though.

see #512

File: docs/src/modules/java/pages/actions.adoc [hunk 3]
```diff
@@ -1,62 +1,104 @@
...
 
+[.tabset]
+Scala::
++
+[source,scala,indent=0]
+.src/main/scala/com/example/fibonacci/FibonacciAction.scala
+----
+include::example$scala-fibonacci-action/src/main/scala/com/example/fibonacci/FibonacciActionGenerated.scala[tag=generated-action]
+----
+<1> Extends the generated `AbstractFibonacciAction`, which extends link:{attachmentsdir}/api/com/akkaserverless/scalasdk/action/Action.html[`Action` {tab-icon}, window="new"].
```
Comment (patriknw) [inline docs/src/modules/java/pages/actions.adoc:69]: api links for Scala should be `{attachmentsdir}/scala-api/`

I added the rsync in the makefile: https://github.com/lightbend/akkaserverless-java-sdk/pull/514

File: samples/scala-fibonacci-action/src/main/proto/com/example/fibonacci/fibonacci.proto [hunk 4]
```diff
@@ -18,15 +18,13 @@ package com.example.fibonacci; // <1>
 
 import "akkaserverless/annotations.proto"; // <2>
 
-option java_outer_classname = "FibonacciApi"; // <3>
```
Comment (patriknw) [inline samples/scala-fibonacci-action/src/main/proto/com/example/fibonacci/fibonacci.proto:21]: 👍

**Review by patriknw — COMMENTED** [review]
looking good, just a few small things...

File: docs/src/modules/java/pages/actions-publishing-subscribing.adoc (hunk 2 above)
Comment (RayRoestenburg) [reply docs/src/modules/java/pages/actions-publishing-subscribing.adoc:38]: ok I'll do the same no worries.

File: docs/src/modules/java/pages/actions.adoc (hunk 3 above)
Comment (RayRoestenburg) [reply docs/src/modules/java/pages/actions.adoc:69]: Thanks

File: docs/src/modules/java/pages/actions-as-controller.adoc (hunk 1 above)
Comment (RayRoestenburg) [reply docs/src/modules/java/pages/actions-as-controller.adoc]: Thanks

**Comment by patriknw** [comment]
"Scala in action" sounds like a book 😄

File: docs/src/modules/java/pages/actions.adoc [hunk 5]
```diff
@@ -1,62 +1,101 @@
...
+<2> Import the Akka Serverless protobuf annotations, or options.
+<3> The protobuf option (akkaserverless.service) is specific to code-generation as provided by the Akka Serverless Maven plugin. This annotation indicates to the code-generation that an Action must be generated.
 
 == Implementing the Action
 
-An Action implementation is a Java class where you define how each message is handled. The class
+An Action implementation is a class where you define how each message is handled. The class
 `FibonacciAction` gets generated for us based on the proto file defined above. Once the
-`FibonacciAction.java` file exist, it is not overwritten, so you can freely add logic to it.
+`FibonacciAction.java` or `FibonacciAction.scala` file exist, it is not overwritten, so you can freely add logic to it.
```
Comment (patriknw) [inline docs/src/modules/java/pages/actions.adoc:41]: Here I think we can use:
```
[.group-java]#`FibonacciAction.java`# [.group-scala]#FibonacciAction.scala#
```

Comment (RayRoestenburg) [reply docs/src/modules/java/pages/actions.adoc:41]: ah great did not know that

**Review by johanandren — APPROVED** [review]

**Pull request merged**